### PR TITLE
Add tooltips to LDAP wizard special attributtes

### PR DESCRIPTION
**Pull request by davitol**
Fixes https://github.com/owncloud/user_ldap/issues/53

@jvillafanez

### CODE REVIEW

File: templates/settings.php [hunk 1]
```diff
@@ -97,9 +97,11 @@
...
-				<p><label for="ldap_quota_def"><?php p($l->t('Quota Default'));?></label><input type="text" id="ldap_quota_def" name="ldap_quota_def" data-default="<?php p($_['ldap_quota_def_default']); ?>" title="<?php p($l->t('in bytes'));?>" /></p>
-				<p><label for="ldap_email_attr"><?php p($l->t('Email Field'));?></label><input type="text" id="ldap_email_attr" name="ldap_email_attr" data-default="<?php p($_['ldap_email_attr_default']); ?>" /></p>
+				<p><label for="ldap_quota_attr"><?php p($l->t('Quota Field'));?></label><input type="text" id="ldap_quota_attr" name="ldap_quota_attr" data-default="<?php p($_['ldap_quota_attr_default']); ?>" title="<?php p($l->t('Leave empty for ownCloud quota. Otherwise, specify an LDAP/AD attribute.'));?>" /></p>
+				<p><label for="ldap_quota_def"><?php p($l->t('Quota Default'));?></label><input type="text" id="ldap_quota_def" name="ldap_quota_def" data-default="<?php p($_['ldap_quota_def_default']); ?>" title="<?php p($l->t('Override ownCloud default quota for LDAP users who do not have a quota set in the Quota Field.
+
```
Comment (jvillafanez) [inline templates/settings.php]: Extra lines are ugly

**Comment by jvillafanez** [comment]
Tested 👍 

It might need a second review for the texts.

File: templates/settings.php [hunk 2]
```diff
@@ -97,9 +97,9 @@
...
-				<p><label for="ldap_quota_def"><?php p($l->t('Quota Default'));?></label><input type="text" id="ldap_quota_def" name="ldap_quota_def" data-default="<?php p($_['ldap_quota_def_default']); ?>" title="<?php p($l->t('in bytes'));?>" /></p>
-				<p><label for="ldap_email_attr"><?php p($l->t('Email Field'));?></label><input type="text" id="ldap_email_attr" name="ldap_email_attr" data-default="<?php p($_['ldap_email_attr_default']); ?>" /></p>
+				<p><label for="ldap_quota_attr"><?php p($l->t('Quota Field'));?></label><input type="text" id="ldap_quota_attr" name="ldap_quota_attr" data-default="<?php p($_['ldap_quota_attr_default']); ?>" title="<?php p($l->t('Leave empty for ownCloud quota. Otherwise, specify an LDAP/AD attribute.'));?>" /></p>
+				<p><label for="ldap_quota_def"><?php p($l->t('Quota Default'));?></label><input type="text" id="ldap_quota_def" name="ldap_quota_def" data-default="<?php p($_['ldap_quota_def_default']); ?>" title="<?php p($l->t('Override ownCloud default quota for LDAP users who do not have a quota set in the Quota Field.'));?>" /></p>
```
Comment (PVince81) [inline templates/settings.php]: never use `ownCloud` in a visible text. Should use the product name variable from the theme.

But here I suggest to just remove "ownCloud" from the text.

**Comment by PVince81** [comment]
@settermjd can you check the wording ?

**Comment by davitol** [comment]
@settermjd Kindly remember

**Comment by davitol** [comment]
Last train to get this merged in oC10 🚆

**Comment by PVince81** [comment]
Let's merge this anyway, even if the wording might not be perfect it's better than nothing.

If changes are required, please submit a separate PR.

**Comment by settermjd** [comment]
Sorry, not sure how I missed this.

**Comment by settermjd** [comment]
fwiw, looks fine.

**Pull request merged**